src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -386,12 +386,7 @@ private void RefreshExpandedDisplay(Action refreshDisplay)
                     {
                         var tree = new WatchTree
                         {
-                            DataContext = new WatchViewModel(),
-
-                            // If Content is 1 string, e.g. "Empty", "null", margin should be more to the left side. 
-                            Margin = newViewModel.Children.Count > 0
```

Comment:
Fixes this bug: 
![image](https://cloud.githubusercontent.com/assets/8158404/12576380/fc985a6c-c41b-11e5-960b-e5802693bb09.png)

Margin should be applied on the content inside treeview, but not on the whole treeview.